src/card/card-actions.jsx

```diff
@@ -18,9 +35,20 @@ let CardActions = React.createClass({
       });
     });
 
+    var expandable;
+    if (this.props.expandable === 'open') {
+      expandable = <OpenIcon/>;
+    } else if (this.props.expandable === 'close') {
+      expandable = <CloseIcon/>;
+    }
+    if (expandable) {
+      expandable = <IconButton style={styles.expandableIconButton} onClick={this._onExpanding}>{expandable}</IconButton>;
+    }
```

Comment:
For long JSX lines, I like to do:
```
expandable = (
  <IconButton
    style={styles.expandableIconButton}
    onClick={this._onExpanding}>
    {expandable}
  </IconButton>
);
```

Also, this `onClick` should be `onTouchTap`.